specification/marketplacecatalog/data-plane/Search/preview/2022-09-25-preview/examples/Search_Get.json

```diff
@@ -0,0 +1,65 @@
+{
+  "parameters": {
+    "searchQuery": "a",
+    "top": 2,
+    "language": "en",
+    "gallery": "Azure",
+    "publishingStage": "Public",
+    "market": "US",
+    "select": [
+      "UniqueProductId",
+      "ProductType"
+    ],
+    "pricingTypes": [
+      "Byol"
+    ],
+    "orderBy": [
+      "Popularity desc",
+      "RatingAverage asc"
+    ],
+    "facets": [
+      "PricingTypes"
```

Comment:
Is casing incorrect? The facet return is pricingTypes (lower case P)

Reply:
fixed